File: Python/1013.py
# O maior

valor = input().split()

a = int(valor[0])
b = int(valor[1])
c = int(valor[2])

maiorAB = (a + b + abs(a - b)) / 2
maiorABC = (maiorAB + c + abs(maiorAB - c)) / 2

print(f"{int(maiorABC)} eh o maior")

File: JavaScript/1013.js
// O maior

var input = require("fs").readFileSync("/dev/stdin", "utf8");
var lines = input.split("\n");

let valor = lines.shift().split(" ");

let a = parseInt(valor[0]);
let b = parseInt(valor[1]);
let c = parseInt(valor[2]);

let maiorAB = (a + b + Math.abs(a - b)) / 2;
let maiorABC = (maiorAB + c + Math.abs(maiorAB - c)) / 2;

console.log(`${parseInt(maiorABC)} eh o maior`);
